src/ADAL.PCL.Desktop/UserPasswordCredential.cs

```diff
@@ -91,5 +91,32 @@ internal override void ApplyTo(DictionaryRequestParameters requestParameters)
 
             SecurePassword = null;
         }
+
+        /// <summary>
+        /// Unused
+        /// </summary>
+        /// <param name="status"></param>
+        public void UnusedMethod(bool status)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://iddp-sonarqube:9000/coding_rules#rule_key=csharpsquid%3AS1186)